### PR TITLE
Add the whole UI

### PR DESCRIPTION
**Pull request by Olamarx**
- React Project
- Add the UI according to the wireframe
- Styled with CSS grid

### CODE REVIEW

File: src/modules/Buttons.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/* eslint-disable array-callback-return */
+import React from 'react';
+
+export default class Buttons extends React.Component {
+  constructor(props) {
```
Comment (topeogunleye) [inline src/modules/Buttons.js]: - [ ] As written in the project requirements, your project should have a directory called `components` and 
- The Calculator.js component should be in the components folder.
- Ensure the `Calculator.js` component is class-based.

File: src/modules/App.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './App.css';
+import Buttons from './Buttons';
+import Upper from './Upper';
+
+// eslint-disable-next-line react/prefer-stateless-function
+export default class App extends React.Component {
```
Comment (topeogunleye) [inline src/modules/App.js]: Great job making your App component class based, however kindly ensure it imports the Calculator.js component after you create the Calculator component

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @Olamarx,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:

> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/App.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return <Calculator />;
+  }
+}
```
Comment (AnselemOdims) [inline src/components/App.js:14]: - [x] [OPTIONAL] As this App is the root component, it will be much better if you put it directly in the src root

**Review by AnselemOdims — APPROVED** [review]
### Hi  @Olamarx 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/App.js (hunk 3 above)
Comment (Olamarx) [reply src/components/App.js:14]: @AnselemOdims thank you, I will do that.

**Pull request merged**